functions/divisional.inc

```diff
@@ -68,11 +68,12 @@ SQL;
             $value = $result->fetch();
         }
     }
-    asort($Departments);
-    asort($Divisions);
 
     // Push non-divisions to the end of the list (force order)
     $endDivs = ['Board Staff', 'Committees', 'Corporate Staff'];
     $Divisions = array_diff($Divisions, $endDivs);
     $Divisions = array_merge($Divisions, $endDivs);
+
+    asort($Divisions);
```

Comment:
Should this one still come before the bit where we push `$endDivs` to the end? It's the departments within a division I was concerned about.

Reply:
Ok change pushed. Re-review please